subjects/concatenate/README.md

```diff
@@ -0,0 +1,14 @@
+## concatenate
+
+### Instructions
+
+Write a program that displays all the arguments as a single string. If there are not enough arguments it should display a new line.
+
+### Usage
+
+```console
```

Comment:
what happens in this case?
Add the explanation in the readme and an example here in usage 

`go run . "hello there" "how are you?"`

`go run . "hello there" "      "`